React v17

Hi!

First of all, thanks for your lib.

```
npm i @eo-locale/react --save
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: react-eo-locale@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from @eo-locale/react@1.4.14
npm ERR! node_modules/@eo-locale/react
npm ERR!   @eo-locale/react@"*" from the root project
```

I suppose it should be something like `"react": ">= 16.8.0"`

--- a/packages/react/package.json
+++ b/packages/react/package.json
@@ -83,8 +83,8 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^16.8.0 || ^17.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0"
   },
   "gitHead": "85f16888e69bd8962e3e20ad92065a6ef5afcb22"
 }

